src/lib/fees/utils.ts

```diff
@@ -0,0 +1,47 @@
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { getEthersProvider } from 'lib/ethersProviderSingleton'
+
+import { FOX_DISCOUNT_DELAY_HOURS } from './parameters'
+
+dayjs.extend(duration)
+const ETHEREUM_AVERAGE_BLOCK_TIME_SECONDS = 12.08
```

Comment:
if we wanna update this automatically we can use alchemy https://docs.alchemy.com/reference/getblocktime